air/src/aux_table/hasher/mod.rs

```diff
@@ -0,0 +1,616 @@
+use crate::utils::{are_equal, binary_not, is_binary, EvaluationResult};
+
+use super::{EvaluationFrame, Felt, FieldElement, HASHER_TRACE_OFFSET};
+use core::ops::Range;
+use vm_core::{
+    hasher::{
+        Hasher, CAPACITY_LEN, DIGEST_LEN, DIGEST_RANGE, NUM_ROUNDS, NUM_SELECTORS, STATE_WIDTH,
+    },
+    utils::range as create_range,
```

Comment:
Thought for the future: I wonder if we should create something like `RangeExt` trait and implement something like `new()` function on it. Might be easier than using this `range()` function.

Reply:
Yeah, I think this makes sense to introduce in a future PR